src/db/table/user_user_type.sql

```diff
@@ -0,0 +1,12 @@
+-- Stores the various user types that each user is
+CREATE TABLE `user_user_type` (
```

Comment:
Because this is an n:m joining table, there's no need for a proxy primary key (what you have as `user_user_type_id`. Instead, the primary key can be the combination of `user_id` and `user_type_id`, eg: ```PRIMARY KEY (`user_id`, `user_type_id`)```. So the only two columns needed in this table is `user_id` and `user_type_id`.